dt: 2.75 #0.75 2.75
n_iter: 20
ns: 20
dist_to_goal: 19 #10 19
v: 0.22
A: 0.5
B: 0.25
C: 0.25


